browser/js/user/user.factory.js

```diff
@@ -6,8 +6,6 @@ app.factory('UserFactory', function($state, $http){
         console.log("Called getUserInfo")
```

Comment:
If a log falls in some code and nobody is around to hear it, does it make a sound?